pkg/util/eventqueue/eventqueue.go

```diff
@@ -0,0 +1,297 @@
+package eventqueue
```

Comment:
Why are we adding another queue implementation, rather than the ones upstream?

I don't know that we need more queues, we need to be using informers and the upstream code, and getting rid of all of these custom queues.